backend/uclapi/dashboard/urls.py

```diff
@@ -11,5 +12,7 @@
     url(r'api/rename/$', rename_app),
     url(r'api/regen/$', regenerate_app_token),
     url(r'api/delete/$', delete_app),
+    url(r'api/setcallbackurl/$', set_callback_url),
+    url(r'api/updatescopes/$', update_scopes),
```

Comment:
I feel like `setcallbackurl` and `updatescopes` should be camelCased, so that they're more obvious to the user

Reply:
These are all internal URLs

Reply:
I think the reason for that may be that we don't have multiple-word urls so far. imo we should have a consistent pattern for that - even for internal URLs. no casing feels like the wrong choice.

Reply:
We do have one multiple-word URL so far: `user/login.callback`; we use this for the dashboard Shibboleth login callback. The other solution has been to use forward slashes, like `tokens/scopes` and `user/data`

Reply:
Can we use one of those here?